scripts/rollup/build.js

```diff
@@ -349,6 +352,9 @@ function getPlugins(
           renaming: !shouldStayReadable,
         })
       ),
+    // HACK to work around the fact that Rollup isn't removing unused, pure-module imports.
+    // Note that this plugin must be called after closure applies DCE.
+    isProduction && stripUnusedImports(pureExternalModules),
```

Comment:
We still need this right?

Reply:
Yup, if we want to avoid a no-op side require being in a production bundle. At least making it a plug-in like @TrySound suggested should be a bit faster since it doesn't have to do any extra file IO